lib/src/connection_fsm.dart

```diff
@@ -12,6 +12,17 @@ abstract class _PostgreSQLConnectionState {
   }
 
   _PostgreSQLConnectionState onMessage(ServerMessage message) {
```

Comment:
Also, since (and if I'm wrong correct me) if NotificationResponseMessages come out of band - i.e. a Query does not trigger them, they are sent to to the user without request - we should wrap their processing in a try-catch block and emit an error on the broadcast stream if something within that processing fails.